rpcs3/rpcs3qt/register_editor_dialog.cpp

```diff
@@ -125,6 +127,7 @@ register_editor_dialog::register_editor_dialog(QWidget *parent, u32 _pc, const s
 			m_register_combo->addItem("SRR0", +SPU_SRR0);
 		}
 
+		m_register_combo->addItem("Reserv Invalidation", +RESERVATION_LOST);
```

Comment:
is this shortened intentionally?

Reply:
I think it would be ugly if its much longer than the other register names so yea.

Reply:
can't you at least put a dot on it?

Reply:
renamed.